beebop/app.py

```diff
@@ -302,13 +308,13 @@ def run_poppunk_internal(sketches: dict,
     job_microreact = q.enqueue(visualise.microreact,
                                args=(p_hash,
                                      fs,
-                                     db_fs,
+                                     full_db_fs,
                                      args,
                                      name_mapping,
                                      species,
                                      redis_host,
                                      queue_kwargs),
-                               depends_on=job_network, **queue_kwargs)
+                               depends_on=Dependency([job_assign, job_network], allow_failure=True), **queue_kwargs)
```

Comment:
I'm surprised you need to specify dependency on job_assign here, when job_network already depends on job_assign..? What difference does allow_failure make?

Reply:
the reason we specify job assign as well is because we need the clusters info which is a dependency... thus if network fails then microreact will run and can still get cluster info as dependency,,,, allow_failiure means we still run if the previous jobs fails